src/binder/clrprivbindercoreclr.cpp

```diff
@@ -159,8 +159,13 @@ HRESULT CLRPrivBinderCoreCLR::BindUsingPEImage( /* in */ PEImage *pPEImage,
                 {
                     if (pCoreCLRFoundAssembly->GetIsInGAC())
                     {
-                        // If we were able to bind to a TPA assembly, then fail the load
```

Comment:
Agreed - we can make it consistent though attempt to load SPC.dll is basically equivalent to axe your own foot.